axes/decorators.py

```diff
@@ -412,15 +494,14 @@ def check_request(request, login_unsuccessful):
                 )
                 attempt.http_accept = \
                     request.META.get('HTTP_ACCEPT', '<unknown>')[:1025]
-                attempt.path_info = \
-                    request.META.get('PATH_INFO', '<unknown>')[:255]
+                attempt.path_info = path_info,path_info
```

Comment:
After double checking, not a typo. Line 497 references 465. 

The `path_info` variable was redefined at the beginning of `check_request`, so that it only gets set once, instead of on every loop iteration.